statusHandler/quotaStatusHandler/printQuotaStatusHandler.go

```diff
@@ -0,0 +1,158 @@
+package quotaStatusHandler
+
+import (
+	"math"
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/logger"
+)
+
+var log = logger.GetOrCreate("statushandler/quotastatushandler")
+
+type quota struct {
+	numReceivedMessages   uint32
+	sizeReceivedMessages  uint64
+	numProcessedMessages  uint32
+	sizeProcessedMessages uint64
+}
+
+//TODO replace this structure with a new one that can output on a route the statistics measured
+//TODO add tests
+// printQuotaStatusHandler implements process.QuotaStatusHandler and is able to periodically print peer connection statistics
+type printQuotaStatusHandler struct {
```

Comment:
I would move this to a different directory and change it's name because the rest of the components in the project with the `statusHandler` suffix implement the `AppStatusHandler` interface, but your component doesn't.

Reply:
+1

Reply:
I find it appropriate to let it together with other "status gathering & processing structs"
Maybe rename the subpackage to "metric" ? Otherwise we will end up having a bunch of subpackages spread all over the place that gathers metrics.

Reply:
It can stay here. It is not critical to implement de AppStatusHandler -